articles/the-role-of-artificial-intelligence-for-smarter-cybersecurity/index.md

```diff
@@ -0,0 +1,53 @@
+Organizations continue to embrace the Internet of Things (IoT), the cloud, and mobile technology. This has influenced considerable changes in the threat landscape and created more vulnerability points. Cybercriminals are leveraging these new vulnerability points to develop and launch sophisticated, high-volume, multi-dimensional attacks. Such attacks mean that data is at risk, and organizations must spend valuable time analyzing potentially malicious files.
+
+Using artificial intelligence software, organizations can process large volumes of threat data and adequately prevent and respond to breaches and hacks. This article will discuss how machine learning and artificial intelligence improve cybersecurity in four important areas: network security, data centers, threat hunting, and vulnerability management.
+
+### Network Security
+
+Network security mainly focuses on the understanding of network topography and creating a security policy of an organization. Securities policies can help to distinguish legitimate network connections from illegitimate ones. But drafting and maintaining policies for multiple networks may be challenging.
+
+Often, some organizations may not have a precise naming convention for workloads and applications. This means that network security teams have to spend significant time attaching different workloads to given applications.
+
+These activities are time-consuming, and the use of artificial intelligence technology can come in handy. AI can expedite these processes by learning and observing traffic patterns and suggesting security policies. This way, organizations save on time, resources, and effort. The organization can allocate these to other areas of technological advancement or development.
+
+### Data Centers
+
+Cyber attackers target [data centers](https://www.cisco.com/c/en/us/solutions/data-center-virtualization/what-is-a-data-center.html) in an attempt to obtain critical data. Cybersecurity has improved over the years with advancements in technology. Being aware of this, cybercriminals regularly develop more advanced malware strains to ensure successful cyber-attacks on organizations&#39; networks.
+
+Organizations can deploy artificial intelligence technology in their data centers to enhance data security. To ensure data security, AI essentially learns normal network behavior. It then detects cyber threats with regard to the variations from the regular network behavior.
```

Comment:
I feel we are describing the role of AI very vaguely here. What are normal patterns of behavior? How are they defined? What type of AI solves this? As a reader, I want to know as much as possible from the article. 

Any person reading this knows about AI being used everywhere. What the reader might lack will be the field-knowledge. If you can talk about detecting shady behavior using an RNN or rule-based expert systems, it adds more value to the reader.